### PR TITLE
Additional feature to process multiple request based on length, 1 Bug Fix, Removed extra imports

### PR DESCRIPTION
**Pull request by Jatish-Khanna**
The compilation was failing on code import caused by missing semi-colon.

In addition to that, additional import statements have been removed

Feature added:
  Where multiple request type can be processed based on the length of the request.
```ini
[request_definition]
r = sub, obj, act
r2 = sub, obj, act, eft

[policy_definition]
p = sub, obj, act, eft

[role_definition]
g = _, _

[policy_effect]
e = priority(p.eft) || deny

[matchers]
m = g(r.sub, p.sub) && r.obj == p.obj && r.act == p.act
```
Note: All 78 test cases have been successfully passed

### CODE REVIEW

File: src/main/java/org/casbin/jcasbin/main/CoreEnforcer.java [hunk 1]
```diff
@@ -424,6 +417,17 @@ public boolean enforce(Object... rvals) {
...
+    private void getRTokens(Map<String, Object> parameters, Object ...rvals) {
+      for(String rKey : model.model.get("r").keySet()) {
+        if(!(rvals.length == model.model.get("r").get(rKey).tokens.length)) { continue; }
+        for (int j = 0; j < model.model.get("r").get(rKey).tokens.length; j ++) {
+          String token = model.model.get("r").get(rKey).tokens[j];
+          parameters.put(token, rvals[j]);
+        }
+   
+      }
+    }
```
Comment (Jatish-Khanna) [inline src/main/java/org/casbin/jcasbin/main/CoreEnforcer.java:430]: This code block access over all the provided request and selects with the one matching the actual request token length.
Earlier, it was selecting just one rule where this adds flexibility for more set of request rule.

File: src/main/java/org/casbin/jcasbin/main/CoreEnforcer.java [hunk 2]
```diff
@@ -332,17 +332,10 @@ public boolean enforce(Object... rvals) {
...
-                    for (int j = 0; j < model.model.get("r").get(rKey).tokens.length; j ++) {
-                      String token = model.model.get("r").get(rKey).tokens[j];
-                      parameters.put(token, rvals[j]);
-                    }
-                    break;
-                  }
-                }
+                
+                // Select the rule based on request size
+                getRTokens(parameters, rvals);
```
Comment (Jatish-Khanna) [inline src/main/java/org/casbin/jcasbin/main/CoreEnforcer.java:338]: Identify the request rule being selected from set of provided rules

**Comment by hsluoyz** [comment]
@tldyl please review.

**Comment by hsluoyz** [comment]
@Jatish-Khanna please resolve conflicts.

**Comment by Jatish-Khanna** [comment]
I am working on it

**Comment by hsluoyz** [comment]
@Jatish-Khanna can you make one PR for one goal? feature, bug fix and style issue should be different PRs.

**Comment by hsluoyz** [comment]
I will close this for now, feel free to open new PRs.

**Pull request closed without merging**